Tests/MockoloTestCase.swift

```diff
@@ -147,9 +136,25 @@ class MockoloTestCase: XCTestCase {
             onCompletion: { ret in
                 let output = (try? String(contentsOf: URL(fileURLWithPath: self.defaultDstFilePath), encoding: .utf8)) ?? ""
                 let outputContents = output.components(separatedBy:  .whitespacesAndNewlines).filter{!$0.isEmpty}
-                let fixtureContents = formattedDstContent.components(separatedBy: .whitespacesAndNewlines).filter{!$0.isEmpty}
-                XCTAssert(fixtureContents == outputContents, "output:\n" + output)
+                let fixtureContents = dstContent.components(separatedBy: .whitespacesAndNewlines).filter{!$0.isEmpty}
+                XCTAssert(outputContents.contains(subArray: fixtureContents), "output:\n" + output)
```

Comment:
Instead of examining all lines, we check whether the output lines contain the fixture.